bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
Ah, I didn't see that due to the wiki being incorrect. I've updated the wiki with the proper types now for JsParseSerialized and JsRunSerialized.